src/sentry/api/endpoints/organization_integration_repository_project_path_configs.py

```diff
@@ -10,11 +11,20 @@
 from sentry.utils.compat import map
 
 
+def gen_regex_field():
```

Comment:
Maybe we should use this for the default branch as well? Even though we pre-populate, still might be good to do

Reply:
@MeredithAnya Yea, we can use the `\w+` regex for the branch name.